src/components/notebook-utils.js

```diff
@@ -42,8 +42,8 @@ export const findPotentialNotebookLockers = async ({ canShare, namespace, wsName
 export const notebookNameValidator = existing => ({
   presence: { allowEmpty: false },
   format: {
-    pattern: /^[^#[\]*?:;@$%+=\\,/]*$/,
-    message: 'can\'t contain these characters: \r \r  @ # $ % * + = ? , [ ] : ; / \\ '
+    pattern: /^[^@#$%*+=?,[\]:;/\\]*$/,
+    message: 'can\'t contain these characters: @ # $ % * + = ? , [ ] : ; / \\ '
```

Comment:
Those `\r`s are strange... I wonder if it was an attempt to make sure the list of characters didn't soft-wrap somewhere in the middle...

Confirmed by looking back at the previous PR: https://github.com/DataBiosphere/terra-ui/pull/1038#discussion_r246189122

It doesn't currently wrap on my screen, but any number of unrelated modifications could change that. Suggestion:

```
    message: h(Fragment, [
      div('Name can\'t contain these characters:'),
      div({ style: { margin: '0.5rem 1rem' } }, '@ # $ % * + = ? , [ ] : ; / \\ ')
    ])
```